provider/app.py

```diff
@@ -158,19 +164,24 @@ def authorizedForTrigger(auth, consumer):
 
 
 def createAndRunConsumer(triggerFQN, params, record=True):
-    if app.config['TESTING'] == True:
-        logging.debug("Just testing")
-    else:
-        # generate a random uuid for new triggers
-        if not 'uuid' in params:
-            params['uuid'] = str(uuid.uuid4())
-
-        consumer = Consumer(triggerFQN, params)
+    # generate a random uuid for new triggers
+    if not 'uuid' in params:
+        params['uuid'] = str(uuid.uuid4())
+
+    # Create a representation for this trigger, even if it is disabled
+    # This allows it to appear in /health as well as allow it to be deleted
+    # Creating this object is lightweight and does not initialize any connections
+    consumer = Consumer(triggerFQN, params)
+    consumers.addConsumerForTrigger(triggerFQN, consumer)
+
+    if 'status' not in params or params['status']['active'] == True:
+        logging.info('{} Trigger was determined to be active, starting...'.format(triggerFQN))
         consumer.start()
-        consumers.addConsumerForTrigger(triggerFQN, consumer)
+    else:
+        logging.info('{} Trigger was determined to be disabled, not starting...'.format(triggerFQN))
```

Comment:
Yes. This is temporary until #160 is finished. At that point, the provider will listen to DB changes directly and will be able to react to changes in status.